packages/bierzo-wallet/src/logic/transactions/BwTransaction.ts

```diff
@@ -0,0 +1,14 @@
+import { Address, BlockchainConnection, ConfirmedTransaction, LightTransaction } from '@iov/bcp';
+import { ReadonlyDate } from 'readonly-date';
+
+export type ParsedTx<T> = Pick<LightTransaction, 'kind'> & { time: ReadonlyDate } & T;
+
+export abstract class BwTransaction<T extends LightTransaction, K> {
```

Comment:
This class is not a "transaction" but a parser. So it should be called something like `BwParser`